helix-term/src/commands.rs

```diff
@@ -3647,6 +3648,36 @@ fn yank(cx: &mut Context) {
     exit_select_mode(cx);
 }
 
+fn yank_joined_impl(editor: &mut Editor, separator: &str, register: char) -> anyhow::Result<()> {
```

Comment:
It looks like the Result here is never used so we can remove the return type from this function